llm-models/embedding/automl/automl_embeddings_example.py

```diff
@@ -0,0 +1,147 @@
+# Databricks notebook source
```

Comment:
You could just put this file in `llm-models/embedding/automl_embeddings_example.py` to reduce nesting, since I don't expect there to be a ton of automl-related notebooks in this repo to warrant its own folder.